### ISSUE TITLE
Unable to import module

### ISSUE THREAD

**Issue by esauter5** [kolodny/redux-create-reducer#6]
Hi,

I noticed in `package.json` it has the `main` file it has `./lib/index.js` as the entry file. However, this folder does not exist. I thought maybe it was part of a build step, but the structure of the npm module is the same. When I change `main` to `./index.js` everything works fine.

If I'm correct here, I'd be happy to submit a PR. Let me know.

**Comment by kolodny** [comment]
yup, you're right, It seems to work as is, but it should be `index.js`. Feel free to open a PR

**Comment by nickhudkins** [comment]
when you say... seems to work as is, do you mean your tests pass? Or you have this integrated into a project and it is working...

**Comment by kolodny** [comment]
Both, see for yourself

```
~ $ cd /tmp/imm
 /tmp/imm $ npm i redux-create-reducer
y@1.0.0 /private/tmp/imm
└── redux-create-reducer@1.1.0  extraneous

npm WARN y@1.0.0 No description
npm WARN y@1.0.0 No repository field.
 /tmp/imm $ node -e 'console.log(  require("redux-create-reducer").createReducer(0, {FOO: () => 123})(null, {type: "FOO"})  )'
123
 /tmp/imm $
```

**Comment by nickhudkins** [comment]
Bizzare 👍  (Not actually bizzare since it is index.js) @esauter5 perhaps RN is doing something silly.

**Comment by esauter5** [comment]
Glad to see it works, but definitely seems like some `index.js` magic to me

**Comment by kolodny** [comment]
OK, published to `1.1.1`. Let me know if that solves it for you

**Comment by esauter5** [comment]
Works great. Thanks!
